deposit/src/main/java/edu/unc/lib/deposit/fcrepo4/IngestContentObjectsJob.java

```diff
@@ -529,4 +536,16 @@ private NodeIterator getChildIterator(Resource resc) {
 	private void addAclProperties(Resource depositResc, Model aipModel) {
 		// TODO add access control properties
 	}
+	
+	private void addDescription(ContentObject obj) {
+		File modsFile = new File(getDescriptionDir(), obj.getPid().getUUID() + ".xml");
+		if (!modsFile.exists()) {
+			return;
+		}
+		try(InputStream modsStream = FileUtils.openInputStream(modsFile)) {
+			obj.addDescription(modsStream);
+		} catch(IOException e) {
+			// just eat the exception, or do we want some exception handling here?
```

Comment:
I figured we didn't want to eat it, but wasn't sure what you had in mind. I added a throws IOException to the header